Update backups.md (#6993)
umbraco/UmbracoDocs: umbraco-cloud/databases/backups.md

Read more about Umbraco Cloud's Backup and data retention policy in the FAQ.
{% endhint %}

## Limitations

> When restoring a database backup on Umbraco Cloud, certain elements could cause issues:

Reviewer: ```suggestion
When restoring a database backup on Umbraco Cloud, certain elements may cause issues:
```

> - **SQL Server logins** - Those could be stored at the server level (in the master database) rather than within the user database itself.

Reviewer: ```suggestion
- **SQL Server logins** - These may be stored at the server level (in the master database) rather than within the user database itself.
```

Author: I am going to rewrite this completetly

> - **SQL Server logins** - Custom SQL Server logins (e.g., admin, sysuser, etc.) may conflict with existing logins when restoring the database in our hosting platform.

Reviewer: ```suggestion
- **SQL Server logins** - Custom SQL Server logins (for example, admin, sysuser, etc.) may conflict with existing logins when restoring the database in our hosting platform.
```

> - **SQL Server logins** - Custom SQL Server logins (for example, admin, sysuser, etc.) may conflict with existing logins when restoring the database in our hosting platform.

Reviewer: ```suggestion
- **SQL Server logins** - Custom SQL Server logins (for example, admin, sysuser, etc.) may conflict with existing logins when restoring the database in the hosting platform.
```

> - **Complex Database Objects** - A complex database object in SQL is an element with external dependencies, special server configurations.

Reviewer: ```suggestion
- **Complex Database Objects** - A complex database object in SQL is an element with external dependencies or special server configurations.
```

> - **Complex Database Objects** - Custom complex database object in SQL is an element with external dependencies or special server configurations which may result in conflicts when restoring the database in our hosting platform.

Reviewer: ```suggestion
- **Complex Database Objects** - Custom complex database objects in SQL is an element with external dependencies or special server configurations, which may result in conflicts when restoring the database in our hosting platform.
```
